### PR TITLE
registry: update supported storage drivers

### PR DESCRIPTION
**Pull request by miminar**
Mention other storage options (like GlusterFS) as well.

Resolves: [bz#1428139](https://bugzilla.redhat.com/show_bug.cgi?id=1428139)

### CODE REVIEW

**Comment by miminar** [comment]
@legionus for technical review.
@mburke5678 FYI.

@ahardin-rh For language review.
This applies to all releases from 3.4 (inclusive) up.

File: install_config/registry/extended_registry_configuration.adoc [hunk 1]
```diff
@@ -303,9 +303,30 @@ are supported:
...
 * link:https://docs.docker.com/registry/storage-drivers/swift/[OpenStack Swift]
 * link:https://docs.docker.com/registry/storage-drivers/gcs/[Google Cloud Storage (GCS)]
+* link:https://docs.docker.com/registry/storage-drivers/azure/[Microsoft Azure]
+* link:https://docs.docker.com/registry/storage-drivers/oss/[Aliyun OSS]
 
 link:https://docs.docker.com/registry/configuration/#maintenance[General registry storage configuration options] are supported.
 
+[NOTE]
+====
+The list includes just the filesystems that need to be configured in the
```
Comment (ahardin-rh) [inline install_config/registry/extended_registry_configuration.adoc]: s/includes just/only includes

File: install_config/registry/extended_registry_configuration.adoc [hunk 2]
```diff
@@ -303,9 +303,30 @@ are supported:
...
+registry's config file with an exception of filesystem which is the default.
+There are other storage options like:
```
Comment (ahardin-rh) [inline install_config/registry/extended_registry_configuration.adoc]: s/config/configuration file 
s/like/including
filesystem, which is

File: install_config/registry/extended_registry_configuration.adoc [hunk 3]
```diff
@@ -303,9 +303,30 @@ are supported:
...
+Storage]
+* xref:../../install_config/persistent_storage/persistent_storage_ceph_rbd.adoc#install-config-persistent-storage-persistent-storage-ceph-rbd[Ceph
+Rados Block Device]
+* xref:../../install_config/persistent_storage/index.adoc#install-config-persistent-storage-persistent-storage-ceph-rbd[And
+more]. See
+xref:../../install_config/storage_examples/index.adoc#install-config-storage-examples-index[examples]
+as well.
+
+Using these storage options requires the
+link:https://docs.docker.com/registry/storage-drivers/filesystem[filesystem
```
Comment (ahardin-rh) [inline install_config/registry/extended_registry_configuration.adoc]: You must configure the link:https://docs.docker.com/registry/storage-drivers/filesystem[filesystem driver] to use these storage options.

**Comment by ahardin-rh** [comment]
@miminar Just a few comments from me. Thanks!

**Comment by mburke5678** [comment]
Incorporating @ahardin-rh 's comments in [4006](https://github.com/openshift/openshift-docs/issues/4006).

**Comment by miminar** [comment]
I'm closing this in favor of #4006.

**Comment by miminar** [comment]
Thank you @ahardin-rh.

**Pull request closed without merging**